src/libraries/Microsoft.PowerFx.Core/Functions/ResourceManager.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Core.Functions
+{
+    public class ResourceManager
```

Comment:
>ResourceManager [](http://example.com/codeflow?start=17&length=15)

This implementation could be delegated to the host and we should only implement IResourceManager
(for tomorrow)

Reply:
storage -> host too